docs/documentation/user.md

```diff
@@ -65,17 +101,17 @@ Tips for adding a new article:
 - Try to put yourself in the shoes of a new Zulip user. What would you want
   to know?
 
+- Remember to explain the purpose of the feature and give context as well
+  as instructions.
```

Comment:
Perhaps extend this sentence to "as well as instructions for how to use or enable it."?